harvester_e2e_tests/integration/test_backup_restore.py

```diff
@@ -124,6 +115,32 @@ def config_backup_target(api_client, conflict_retries, backup_config, wait_timeo
 
     yield spec
 
+    # remove unbound LH backupVolumes

```

Comment:
Since main parts of this PR is doing refactor.
For legacy backupvolume cleanup, please just refer line 118 to 143.